terraform/cloud-platform-components/fluentd.tf

```diff
@@ -0,0 +1,29 @@
+resource "helm_release" "fluentd_es" {
+  name  = "fluentd-es"
+  chart = "../../helm-charts/fluentd-es"
+
+  set {
+    name  = "namespace"
+    value = "logging"
+  }
+
+  set {
+    name  = "image.tag"
```

Comment:
This is not required, we are using the default value (and in case of an upgrade should upgrade the default in the chart).